api.md

```diff
@@ -266,71 +266,157 @@ Output:
 
 Status Code: 200 or 400
 
+### Create Project
+
+POST "/project_create"
+
+This will create a project with the current user as the admin.
+
+Only requires name and description of the project.
+
+Input: A JSON body with the following **required** parameters.
+
+```typescript
+{
+    name: string;
+    description: string;
+}
+```
+
+Status Code: 201 or 400
+
+### Get Project
+
+GET "/project_get"
+
+This will get the project's name, description and creation time.
+
+Input: A JSON body with the following **required** parameters.
+
+```typescript
+{
+    projectid: string;
+}
+```
+
+Status Code: 200 or 400
+
+### Create Task
+
+POST "/task_create"
```

Comment:
-> task_create has to return the task's id as well
-> needs to take in deadline as well
-> maybe rename users to assignedTo for consistency